src/applications/differential/storage/DifferentialRevision.php

```diff
@@ -1127,7 +1127,13 @@ public function getFieldValuesForConduit() {
       'color.ansi' => $status->getANSIColor(),
     );
 
+    $stack_graph = id(new DifferentialRevisionGraph())
+        ->setSeedPHID($this->getPHID())
+        ->setLoadEntireGraph(true)
+        ->loadGraph();
+
     return array(
+      'depends_on' => $stack_graph->getEdges(DifferentialRevisionDependsOnRevisionEdgeType::EDGECONST,),
```

Comment:
```suggestion
      'dependsOn' => $stack_graph->getEdges(DifferentialRevisionDependsOnRevisionEdgeType::EDGECONST,),
```

To match same case-style as the rest of the fields.